lib/handlers/health-check.ts

```diff
@@ -52,6 +55,12 @@ export class HealthCheckHandler {
          */
         await this.compilationQueue.enqueue(async () => {}, {highPriority: true});
 
+        if (!this.compilerHandler.hasLanguages()) {
```

Comment:
Need to double check if we can assume all the compilers could theoretically have been loaded at this point. Healthcheck request might arrive before they're initialized?